README.md

```diff
@@ -165,6 +165,25 @@ Take the following steps to launch applications from sdl core.
   6. Both applications should show up on the head unit for launching
   7. Select the other application, and you should see it launched and brought to the foreground on the phone
 
+### Test Covarage
+Used technologies :
+  * GCOV - is a test coverage program.
+  * LCOV - is a graphical front-end for GCC's coverage testing tool for gcov.
+  * codecov.io - is service for assembling code coverage and represent in form clear for reading.
+
+Excluded folders :
```

Comment:
@Kozoriz, on my opinion this is quite important block it requires more info.
Here should be the info why we cover only src.

I think it's better to wrap patterns and command in the "How to get Test Coverage localy" not with quotes since they could be processed by markdown formatter. Better to wrap with "`" to prevent processing.